cypress/support/e2e.js

```diff
@@ -14,7 +14,7 @@
 // ***********************************************************
```

Comment:
Let's use "ts" extension for all files. And please remove the comments, you can always check the docs and understand how to use this file.

Reply:
Fixed